sigs.yaml

```diff
@@ -823,6 +840,38 @@ sigs:
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://docs.google.com/document/d/1B0NmmKVh8Ea5hnNsbUsJC7ZyNCsq_6NXl5hRdcHlJgY/edit?usp=sharing
       recordings_url: https://www.youtube.com/playlist?list=PLutJyDdkKQIpOT4bOfuO3MEMHvU1tRqyR
+    - description: Regular VMware Subproject Meeting
```

Comment:
This can be (removed|migrated to UG) as we already have discrete meetings for all the subprojects

Reply:
Not sure I follow, this change moves the existing SIG VMware biweekly meeting into a subproject meeting. Is the VMware subproject meetings defined somewhere else?

Reply:
the cloud-provider-vsphere subproject has its own monthly meeting (different schedule than the SIG VMware biweekly), cluster-api-provider-vsphere has its own biweekly meeting (again, different schedule from SIG VMware)

Reply:
These meetings were moved as part of kubernetes/community#3886

Reply:
ah, I see now. Will fix tha